src/CSET/operators/plot.py

```diff
@@ -429,6 +429,131 @@ def _plot_and_save_vertical_line_series(
     plt.close(fig)
 
 
+def _plot_and_save_histogram_series(
+    cube: iris.cube.Cube, filename: str, title: str, vmin: float, vmax: float, **kwargs
```

Comment:
Can we make the histtype a configurable option? In some circumstances a bar will be nice to view, but in others the line would be easier, especially if having multiple lines on one plot (i.e. for different models).

Reply:
I have added a configurable histogram_type option to the rose-suite.conf and the plot. It is preset to a lineplot.

Reply:
change 8ccc6d9..fd08bc0